### PR TITLE
Fix Swallowed Errors

### PR DESCRIPTION
**Pull request by alrs**
I found some places in rack where error variables are being collected but not used.

### CODE REVIEW

File: api/cmd/build/clean.go [hunk 1]
```diff
@@ -56,6 +56,9 @@ func clean() {
 	r.FieldsPerRecord = 4
 
 	records, err := r.ReadAll()
+	if err != nil {
+		fmt.Printf("WARNING: %s\n", err)
```
Comment (MiguelMoll) [inline api/cmd/build/clean.go]: There should be a return after the warning so the code doesn't continue in an unknown state.

**Comment by MiguelMoll** [comment]
This is a great PR. Thanks for this. 👍

File: api/cmd/build/clean.go (hunk 1 above)
Comment (alrs) [inline api/cmd/build/clean.go]: `return` added.

**Comment by MiguelMoll** [comment]
[Fix Swallowed Errors](https://trello.com/c/N1cnpIOS/2667-fix-swallowed-errors)

**Pull request closed without merging**